### PR TITLE
Update copy changes on /download/server/linuxone for 16.04

### PR DESCRIPTION
**Pull request by barrymcgee**
## Done

Update copy on /download/server/linuxone for 16.04

## QA

Cross ref changes with copy doc

## Issue / Card

Copy doc: https://docs.google.com/document/d/1d85IfqqxZYYZ0orj5lBhqLWLGpLYWRLdvs75lYsplGs/edit

### CODE REVIEW

**Comment by barrymcgee** [comment]
[/download/linuxone updates](https://trello.com/c/DhsDrfnw/1876-download-linuxone-updates)

File: templates/download/server/linuxone.html [hunk 1]
```diff
@@ -23,10 +23,10 @@
             <div class="twelve-col no-margin-bottom">
```
Comment (anthonydillon) [inline templates/download/server/linuxone.html]: Page title should be "Download IBM LinuxONE and z Systems"

Comment (anthonydillon) [inline templates/download/server/linuxone.html]: Page tags should be: "IBM, LinuxONE, Ubuntu Server, OpenStack, Cloud, Ubuntu OpenStack, z Systems, Mainframe"

File: templates/download/server/linuxone.html [hunk 2]
```diff
@@ -23,10 +23,10 @@
...
 
-                        <p>Ubuntu {{lts_release_full_with_point}} for IBM LinuxONE and z&nbsp;Systems is priced at $19,500 per drawer per year, covering unlimited usage of Ubuntu within that drawer, 24x7 Ubuntu Advantage Advanced support services and regular security and reliability fixes from Canonical.</p>
+                        <p>Ubuntu {{lts_release_full}} for IBM LinuxONE and z&nbsp;Systems is priced at $19,500 per drawer per year, covering unlimited usage of Ubuntu within that drawer, 24x7 Ubuntu Advantage Advanced support services and regular security and reliability fixes from Canonical.</p>
 
                         <p>Your use of Ubuntu on z&nbsp;Systems and LinuxONE in production constitutes <a href="http://www.ubuntu.com/legal/short-terms">acceptance of these terms</a>.</p>
```
Comment (anthonydillon) [inline templates/download/server/linuxone.html]: Should be relative link.

**Pull request merged**